_pages/author-shubhra.html

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This page already got added in a previous pull request #105 
It can be seen live here now: https://leomvit.com/author-shubhra-dixit

You can delete this file and edit the one present in the repository if you'd like to change something. Also I'm seeing that your photo isn't visible so that's one fix needed.